### PR TITLE
fix/Mexc - Unable to submit close market order due to TypeError: unsupported operand type(s)

### PR DESCRIPTION
**Pull request by Jbekker**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:
Fixes issue #7090, also added a unit test for this scenario.

**Tests performed by the developer**:
Reproduced the issue with a unit test.

**Tips for QA testing**:

### CODE REVIEW

File: hummingbot/connector/exchange/mexc/mexc_exchange.py [hunk 1]
```diff
@@ -189,13 +189,13 @@ async def _place_order(self,
             price_str = f"{price:f}"
             api_params["price"] = price_str
         else:
-            if trade_type.name.lower() == 'buy':
+            if trade_type == TradeType.BUY:
```
Comment (cardosofede) [inline hummingbot/connector/exchange/mexc/mexc_exchange.py:192]: 🙌

**Review by cardosofede — APPROVED** [review]
good catch, pls @rapcmia test and merge after it!

**Comment by rapcmia** [comment]
PR update:
- Run test locally and setup market making pmm_simple controller 
- We confirm that the issue reported has been fixed regarding`TypeError: unsupported operand type(s)` 
- However there was an instance on the test where it was stuck on `waiting for close order to be filled..` when the market order was completed successfully
- A separate ticket created https://github.com/hummingbot/hummingbot/issues/7129 since its not related to this PR

**Review by rapcmia — APPROVED** [review]

**Comment by rapcmia** [comment]
Merged to development and will be part of release version 2.0.1

**Pull request merged**